x-pack/plugins/infra/public/alerting/common/components/alert_preview.tsx

```diff
@@ -271,33 +263,27 @@ export const AlertPreview: React.FC<Props> = (props) => {
                 iconType="alert"
               >
                 {previewError.body && (
-                  <FormattedMessage
-                    id="xpack.infra.metrics.alertFlyout.alertPreviewErrorDesc"
-                    defaultMessage="Try again later, or {viewTheError}."
-                    values={{
-                      viewTheError: <EuiLink onClick={onOpenModal}>view the error</EuiLink>,
-                    }}
-                  />
+                  <>
+                    <FormattedMessage
+                      id="xpack.infra.metrics.alertFlyout.alertPreviewErrorDesc"
+                      defaultMessage="Please try again later or see details for more information."
+                    />
+                    <EuiSpacer size={'s'} />
+                    <EuiAccordion
+                      id="alertErrorDetailsAccordion"
+                      buttonContent={i18n.translate(
+                        'xpack.infra.metrics.alertFlyout.errorDetails',
+                        {
+                          defaultMessage: 'Details',
+                        }
+                      )}
+                    >
+                      <EuiCodeBlock>{previewError.body.message}</EuiCodeBlock>
```

Comment:
Add `<EuiSpacer size={'s'} />` before the codeblock so that there is space between the codeblock and accordion toggle. You may need to wrap this in a fragment.